src/js/themes/hpe.js

```diff
@@ -285,10 +286,17 @@ export const hpe = deepFreeze({
         primary &&
         colorValue &&
         `:hover {
+        color: ${
+          backgroundIsDark(colorValue, theme)
+            ? theme.global.colors['text-strong'].dark
+            : theme.global.colors['text-strong'].light
+        };
         background-color: ${
           theme.dark
-            ? theme.global.colors[colorValue].dark
-            : theme.global.colors[colorValue].light
+            ? theme.global.colors[colorValue].dark ||
+              theme.global.colors[colorValue]
```

Comment:
had to account for if a user passes in `purple!` that doesn't have a `light` or `dark`